src/serac/physics/utilities/variational_form/array.hpp

```diff
@@ -80,10 +135,16 @@ constexpr auto rotate(const array<T, n>& original, int rotation)
   return rotated;
 }
 
+/// @cond
 constexpr int orientation(const array<int, 2> a, const array<int, 2> b) { return (a[0] == b[0]) ? 1 : -1; }
+/// @endcond
 
+// NOTE: Currently not used anywhere
+/**
+ * @brief
+ */
 template <int n>
-constexpr auto orientation(const array<int, n> a, const array<int, n> b)
+constexpr auto orientation(const array<int, n>& a, const array<int, n>& b)
```

Comment:
I believe this function was supposed to help "undo" a rotation (defined a few lines above). So, if it was the case that `auto b = rotate(a, 3)`, then `orientation(a, b)` would return either 3 or -3 (or something like that). We can also just delete this here, as it's unused.

Originally, this was used for detecting orientations of shared faces between Nedelec elements, to figure out the appropriate local rotation to apply to them.